include/translate-option.php

```diff
@@ -40,7 +40,7 @@ class PLL_Translate_Option {
 	/**
 	 * Cache for the translated values.
 	 *
-	 * @var PLL_Cache
+	 * @var PLL_Cache<mixed>
```

Comment:
```suggestion
	 * @var PLL_Cache<array|string>
```
According to the result of `self::translate_string_recursive()`.